skimage/measure/fit.py

```diff
@@ -492,66 +488,76 @@ def estimate(self, data):
         success : bool
             True, if model estimation succeeds.
 
-        """
 
+        References:
+        -----------
+            [1] Halir, R., Flusser, J.: 'Numerically Stable Direct Least Squares
+                Fitting of Ellipses'
```

Comment:
Use `.. [1] Halir, R.` etc., with no indent relative to the heading. See other docstrings in the library for examples.